src/util.ts

```diff
@@ -393,3 +393,27 @@ const addGraphTotals = (
   )
   return { totalTxs, totalUsd }
 }
+
+export const movingAveDataSort = (
+  data: Data[]
+): Array<{ date: string; allTxs: number }> => {
+  const aveArray: Array<{ date: string; allTxs: number }> = []
+  if (Array.isArray(data) === false) return aveArray
+  const dataLength = data.length
+  let sevenDaySum: number = 0
```

Comment:
Move line 403 inside the `for` loop on line 404. Then you'll also be able to delete line 416. I believe `let` and `const` are limited to the scope of the block statement, so each iteration will create a new `sevenDaySum` that will be equal to 0 (zero).

Reply:
Changed according to comment.